cypress/e2e/views/tags.view.ts

```diff
@@ -24,7 +24,7 @@ export const nameInput = "input[name=name]";
 export const rankInput = "input[aria-label='rank']";
 export const rankHelper = "div[id=rank-helper]";
 export const tagTable = "table[aria-label='tag-table']";
-export const tagMenuButton = "button[aria-label='Actions']";
+export const tagMenuButton = 'button[aria-label="Kebab toggle"]';
```

Comment:
Can you please move it to view file?

Reply:
This selector is already under tags.view

Reply:
oh. sorry, that is view file indeed :-)